gdal/gcore/gdal_misc.cpp

```diff
@@ -79,6 +89,34 @@ static int GetMinBitsForPair(const bool pabSigned[], const int panBits[])
     return std::max(panBits[0], panBits[1]);
 }
 
+int GetDataTypeElementSizeBits( GDALDataType eDataType )
```

Comment:
Missing static qualifier.
Will fix "gdal_misc.cpp:92:5: error: no previous prototype for function 'GetDataTypeElementSizeBits' [-Werror,-Wmissing-prototypes]"